Sources/PointFree/Pricing.swift

```diff
@@ -499,23 +505,28 @@ let extraSpinnerStyles =
     <> (input & .elem(.other("::-webkit-inner-spin-button"))) % opacity(1)
     <> (input & .elem(.other("::-webkit-outer-spin-button"))) % opacity(1)
 
-private let pricingFooterView = View<(Database.User?, Bool)> { currentUser, expand in
+private let pricingFooterView = View<(Database.User?, PricingFormFields, Route?)> { currentUser, formFields, route in
   gridRow([Styleguide.class([Class.pf.colors.bg.white])], [
     gridColumn(sizes: [.mobile: 12], [], [
       div(
         [Styleguide.class([Class.padding([.mobile: [.top: 2, .bottom: 3]])])],
         currentUser
-          .map(const(stripeForm.view(expand)))
-          ?? [gitHubLink(text: "Sign in with GitHub", type: .black, redirectRoute: .pricing(nil, expand: expand))]
-        )
+          .map(const(stripeForm.view(formFields)))
+          ?? [
+            gitHubLink(
+              text: "Sign in with GitHub",
+              type: .black,
+              redirectRoute: route ?? .pricing(nil, expand: false)
```

Comment:
I think this means it will take people back to the homepage. seems more ideal to take them to the subscription page.

Reply:
and I mean it'll go to the homepage in the situation that this view is reused on the homepage.

we could also stop using it on the homepage.

Reply:
I don't think `homeView` passes `route` along, so it should do pricing instead, no? I only updated the pricing views to pass `route` along so that they redirect to the same pricing page.